Automattic/wp-calypso
# Automattic for Agencies: Redesign the Referrals page

Resolves https://github.com/Automattic/jetpack-genesis/issues/333

## Proposed Changes

This PR redesigns the Referrals page.

NOTE: Please ignore the status badge for now. This will be implemented in another PR.

## Testing Instructions

1) Open the A4A live link.
2) Go to Referrals > Verify that you can see the UI as shown below:

<img width="1615" alt="Screenshot 2024-04-25 at 11 04 31 AM" src="https://github.com/Automattic/wp-calypso/assets/10586875/3126ec07-ca80-4c0f-8497-255fb4026b12">

3) Go to /sections/referrals/primary/referrals-overview/index.tsx > Change the `hasPayeeAccount` to true  > Refresh the page > Verify that you can see the UI as shown below. Notice the button.

<img width="967" alt="Screenshot 2024-04-25 at 11 44 16 AM" src="https://github.com/Automattic/wp-calypso/assets/10586875/6ee3c054-4aff-4b80-91a2-fe5891d70223">

Other screen sizes:

<img width="357" alt="Screenshot 2024-04-25 at 4 17 21 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/1bc578a6-9d1b-4005-bc8f-4e396fa07ff8">

<img width="760" alt="Screenshot 2024-04-25 at 4 17 33 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/9044463a-9ee4-44c7-8895-28698138e5db">



## Pre-merge Checklist
